lms/templates/onedrive.html.jinja2

```diff
@@ -0,0 +1,5 @@
+{% extends "lms:templates/base.html.jinja2" %}
+
+{% block scripts %}
+  <script type="text/javascript" src="https://js.live.net/v7.2/OneDrive.js"></script>
+{% endblock %}
```

Comment:
Ok, so we've got our backend rendering an HTML page with `OneDrive.js` in it.

Should this page be extending `base.html.jinja2`? The base template contains a bunch of stuff that I'm not sure whether we want from this page.

Reply:
Good point, changed to a minimal html template, no base.